cypress/e2e/awx/resources/workflowJobTemplates.cy.ts

```diff
@@ -208,17 +415,83 @@ describe('Workflow Job Templates Tests', () => {
       cy.deleteAwxOrganization(organization, { failOnStatusCode: false });
     });
 
-    it.skip('can delete a workflow job template from the details page', () => {
-      //Utilize a workflow job template created in the beforeEach block
-      //Assert original values on the WFJT
-      //After delete, intercept the API response and assert the deletion
+    it('can bulk delete multiple workflow job templates from the list toolbar', () => {
+      let newWorkflowJobTemplate: WorkflowJobTemplate;
+      cy.createAwxWorkflowJobTemplate({
+        organization: organization.id,
+        inventory: inventory.id,
+      }).then((wfJT) => {
+        newWorkflowJobTemplate = wfJT;
+        cy.navigateTo('awx', 'templates');
+        cy.filterTableByMultiSelect('name', [
+          workflowJobTemplate.name,
+          newWorkflowJobTemplate?.name,
+        ]);
+        cy.selectTableRowByCheckbox('name', workflowJobTemplate.name, { disableFilter: true });
+        cy.selectTableRowByCheckbox('name', newWorkflowJobTemplate.name, { disableFilter: true });
+        cy.clickToolbarKebabAction('delete-selected-templates');
+        cy.clickModalConfirmCheckbox();
+        cy.intercept(
+          'DELETE',
+          awxAPI`/workflow_job_templates/${workflowJobTemplate.id.toString()}/`
+        ).as('deleteWFJT');
+        cy.intercept(
+          'DELETE',
+          awxAPI`/workflow_job_templates/${newWorkflowJobTemplate.id.toString()}/`
+        ).as('deleteWFJT2');
+        cy.clickModalButton('Delete template');
+        cy.wait('@deleteWFJT')
+          .its('response')
+          .then((response) => {
+            expect(response?.statusCode).to.eql(204);
+          });
+        cy.wait('@deleteWFJT2').then((deletedWFJT2) => {
+          expect(deletedWFJT2?.response?.statusCode).to.eql(204);
+        });
+        cy.assertModalSuccess();
+        cy.clickButton(/^Close$/);
+        cy.clearAllFilters();
+      });
+    });
+  });
+
+  describe('Workflow Job Templates: Delete without using org and inventory', function () {
+    let workflowJobTemplate: WorkflowJobTemplate;
+    let organization: Organization;
+    let inventory: Inventory;
+    before(function () {
+      organization = this.globalOrganization as Organization;
```

Comment:
we want to move away from relying on previously created resources like `Global Organization` because we've seen issues with this. Please create a new organization resource instead of using Global Organization. Thanks!